### PR TITLE
DWR-491, denormalize grade_code (in reporting data mart)

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: migrate-reporting/src/main/resources/application.staging.to.reporting.sql.yml [hunk 1]
```diff
@@ -433,19 +433,20 @@ sql:
         exam:
           sql:
             insert: >-
-              INSERT INTO reporting.exam (id, grade_id, student_id, school_id, iep, lep, section504, economic_disadvantage,
-                                          migrant_status, eng_prof_lvl, t3_program_type, language_code, prim_disability_type,
-                                          type_id, school_year, asmt_id, asmt_version, opportunity, completeness_id,
-                                          completeness_code, administration_condition_id, administration_condition_code,
-                                          session_id, performance_level, scale_score, scale_score_std_err, import_id,
-                                          completed_at,
+              INSERT INTO reporting.exam (id, grade_id, grade_code, student_id, school_id, iep, lep, section504,
```
Comment (malaffoon) [inline migrate-reporting/src/main/resources/application.staging.to.reporting.sql.yml]: this just adds `grade_code`, everything else is cleaning up line length wrapping

File: migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/step/UpsertAsmtsIT.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public void itShouldDoNothingIfBatchDoesNotHaveRightImportContent() {
 
         getStepExecutionContext().put("batch", new MigrateBatch(mock(Migrate.class), newArrayList(ImportContent.CODES)));
 
-        JobExecution jobExecution = launchStep(upsertAsmtsStepName);
+        final JobExecution jobExecution = launchStep(upsertAsmtsStepName);
```
Comment (malaffoon) [inline migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/step/UpsertAsmtsIT.java]: unrelated, fix warning

File: migrate-reporting/src/test/resources/WarehouseEntitiesTearDown.sql [hunk 3]
```diff
@@ -17,15 +17,15 @@ DELETE FROM warehouse_test.student where id in (-11,  -89, -88, -87, -86, -33);
 DELETE FROM warehouse_test.student_group where id in ( -91, -8, -7);
 
 -- ------------------------------------------ School/Districts --------------------------------------------------------------------------------------------------
-DElETE FROM warehouse_test.school where id in (-99, -98, -1);
```
Comment (malaffoon) [inline migrate-reporting/src/test/resources/WarehouseEntitiesTearDown.sql]: unrelated, it was driving me nuts  ;)

Comment (agorina) [inline migrate-reporting/src/test/resources/WarehouseEntitiesTearDown.sql]: That was the test, you passed :)

**Pull request merged**